File: pe65.py
def sumdig(n):
    s=0
    while(n>0):
        s += n%10
        n//=10
    return s

num=0
den=1
for i in range(100,1,-1):
    if(i%3 ==0):
        a = 2*(i//3)
    else:
        a=1

    num = num + a*den
    t = num
    num = den
    den = t
print(sumdig(num+den*2))

File: pe65.cpp
#include<iostream>

using namespace std;

int main()
{
	unsigned long long int num=0;
	unsigned long long int den =1;
	for(int i=100;i>1;i--)
	{
		unsigned long long int a;
		if(i%3 == 0)
			a=2* (i/3);
		else
			a=1;
		
		num = num + a*den;
		unsigned long long int t =num;
		num = den;
		den= t;
	}
	cout<<num+den*2;
}
